__device-tests__/gutenberg-editor-block-insertion.test.js

```diff
@@ -38,6 +39,28 @@ describe( 'Gutenberg Editor tests for Block insertion', () => {
 		editorPage = new EditorPage( driver );
 	} );
 
+	afterEach( async () => {
+		let blockExist = await editorPage.hasBlockAtPosition( 1 );
```

Comment:
This block of code is duplicated across the tests that you used them in. Might be best if we put it into a function and call it, maybe `utils.js` could be a could spot, let me know what you think? Also is there a reason it's not used in the other tests?

Reply:
I get what you're saying and it makes sense. Do you think its better if the code goes in utils or editor-page? The code uses functions from editor-page, if it goes in utils I would have to import from editor page.